src/styles/_catalog.scss

```diff
@@ -1,22 +1,30 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+@import "_mixins.scss";
 
-  max-width: 944px;
-  margin: 0 auto;
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, 25%);
```

Comment:
check mobile breakpoint
looks like bug is here
![Screenshot from 2023-03-13 12-15-10](https://user-images.githubusercontent.com/33029864/224672907-dc3b349a-8acd-4ead-b058-da0407e41ad3.png)